avm/res/storage/storage-account/tests/e2e/max/main.test.bicep

```diff
@@ -250,8 +250,6 @@ module testDeployment '../../../main.bicep' = [
             metadata: {
               testKey: 'testValue'
             }
-            enableWORM: true
-            WORMRetention: 666
```

Comment:
I take it they don't exist? At least I didn't find them...

Reply:
They don't, apparently it is allowed to use non existing properties. Not ideal and wondering how many cases like this we have in our examples

Reply:
It is. Usually the only indicator are scribe lines